libpod/runtime_ctr.go

```diff
@@ -460,3 +465,27 @@ func (r *Runtime) GetLatestContainer() (*Container, error) {
 	}
 	return ctrs[lastCreatedIndex], nil
 }
+
+// reserveLabels walks the list o fcontainers and reserves the label, so new containers will not
+// get them.
+func (r *Runtime) reserveLabels() error {
+	containers, err := r.state.AllContainers()
+	if err != nil {
+		return err
+	}
+	for _, ctr := range containers {
+		label.ReserveLabel(ctr.ProcessLabel())
+	}
+	return nil
+}
+
+// InitLabels allocates an new label to return to the caller
+func (r *Runtime) InitLabels(labelOpts []string) (string, string, error) {
```

Comment:
Does this still need to be public? I think we can make this initLabels() instead.